src/Xamarin.Android.Build.Tasks/Tasks/Lint.cs

```diff
@@ -107,62 +107,62 @@ public class Lint : AndroidToolTask
 		/// issue id's or categories.
 		/// </summary>
 		/// <value>The list of issues to check.</value>
-		public string CheckIssues { get; set; }
+		public string? CheckIssues { get; set; }
 
 		/// <summary>
 		/// Add the given folders (or paths) as a resource
 		/// directories for the project.
 		/// </summary>
 		/// <value>An array of ITaskItems containing the resource directories.</value>
-		public ITaskItem[] ResourceDirectories {get; set;}
+		public ITaskItem[]? ResourceDirectories {get; set;}
 
 		/// <summary>
 		/// Add the given folders (or paths) as a source directories
 		/// for the project.
 		/// </summary>
 		/// <value>An array of ITaskItems containing the source directories.</value>
-		public ITaskItem[] SourceDirectories   {get; set;}
+		public ITaskItem[]? SourceDirectories   {get; set;}
 
 		/// <summary>
 		/// Add the given folder (or path) as a class directories
 		/// for the project.
 		/// </summary>
 		/// <value>An array of ITaskItems containing class directories.</value>
-		public ITaskItem[] ClassDirectories    {get; set;}
+		public ITaskItem[]? ClassDirectories    {get; set;}
 
 		/// <summary>
 		/// Add the given folders (or jar files, or paths) as
 		/// class directories for the project.
 		/// </summary>
 		/// <value>An array of ITaskItems containing the class path jars.</value>
-		public ITaskItem[] ClassPathJars       {get; set;}
+		public ITaskItem[]? ClassPathJars       {get; set;}
 
 		/// <summary>
 		/// Add the given folders (or paths) as a
 		/// class libraries for the project.
 		/// </summary>
 		/// <value>An array of ITaskItems containing the list of directories</value>
-		public ITaskItem[] LibraryDirectories  {get; set;}
+		public ITaskItem[]? LibraryDirectories  {get; set;}
 
 		/// <summary>
 		/// Add the given jar files as a
 		/// class library for the project.
 		/// </summary>
 		/// <value>An array of ITaskITems containing the list of .jar files</value>
-		public ITaskItem[] LibraryJars         {get; set;}
+		public ITaskItem[]? LibraryJars         {get; set;}
 
 		protected override string ToolName {
 			get { return OS.IsWindows ? "lint.bat" : "lint"; }
 		}
 
 		public Lint ()
 		{
-			ResourceDirectories = Array.Empty<ITaskItem> ();
-			SourceDirectories = Array.Empty<ITaskItem> ();
-			ClassDirectories = Array.Empty<ITaskItem> ();
-			ClassPathJars = Array.Empty<ITaskItem> ();
-			LibraryDirectories = Array.Empty<ITaskItem> ();
-			LibraryJars = Array.Empty<ITaskItem> ();
+			ResourceDirectories = [];
+			SourceDirectories = [];
+			ClassDirectories = [];
+			ClassPathJars = [];
+			LibraryDirectories = [];
+			LibraryJars = [];
 		}
```

Comment:
Can we remove this `ctor` completely, and define all these inline, such as:
```
public ITaskItem[] LibraryJars         {get; set;} = [];
```
Then they don't need to be nullable either?